package/kedro_viz/server.py

```diff
@@ -130,7 +131,7 @@ def run_server(
     from kedro.framework.startup import bootstrap_project
 
     parser = argparse.ArgumentParser(description="Launch a development viz server")
-    parser.add_argument("project_path", help="Path to a Kedro project")
+    parser.add_argument("--project-path", help="Path to a Kedro project")
```

Comment:
For dev mode, earlier we had project_path as a required argument. This is good to make it consistent with cli, but can we add a default similar to cli ?